plugins/beacon/jvm/README.md

```diff
@@ -2,4 +2,4 @@
 
 ## Gradle Dependency
 
-`implementation("com.intuit.player.jvm.plugins:beacon:$VERSION")`
+`implementation("com.intuit.player.plugins:beacon:$VERSION")`
```

Comment:
We should either link to the docs for this plugin, or maybe even figure out how we could put the info in this file and load it in the doc site one?